wcfsetup/install/files/acp/templates/systemCheck.tpl

```diff
@@ -104,6 +104,25 @@
 			<small>{lang}wcf.acp.systemCheck.php.sha256.description{/lang}</small>
 		</dd>
 	</dl>
+	
+	<dl{if !$results[php][gd][result]} class="formError"{/if}>
+		<dt>{lang}wcf.acp.systemCheck.php.gd{/lang}</dt>
+		<dd>
+			{if $results[php][gd][result]}
+				{@$statusOk} {lang}wcf.acp.systemCheck.pass{/lang}
+			{else}
+				<ul class="nativeList">
+					{if !$results[php][gd][jpeg]}
+						<li>{@$statusInsufficient} <kbd>jpeg</kbd></li>
```

Comment:
Why are you using `<kdb>` here but quotation marks in `wcf.acp.systemCheck.php.gd.description`? (Same applies to the next `<li>`.)

Reply:
I kept it consistent to the list of missing PHP extensions.